chef/cookbooks/tempest/templates/default/tempest.conf.erb

```diff
@@ -111,6 +111,33 @@ lock_path = /var/run/tempest
 #publish_errors = false
 
 
+[auth]
+
+#
+# From tempest.config
+#
+
+# Allows test cases to create/destroy tenants and users. This option
+# requires that OpenStack Identity API admin credentials are known. If
+# false, isolated test cases and parallel execution, can still be
+# achieved configuring a list of test accounts (boolean value)
+# Deprecated group/name - [compute]/allow_tenant_isolation
+# Deprecated group/name - [orchestration]/allow_tenant_isolation
+allow_tenant_isolation = true
```

Comment:
Yep, it fixes it. can you please revert to the default = false to make this PR mergable?